tests/capi/meson.build

```diff
@@ -12,7 +12,7 @@ unittest_capi_inference_latency = executable('unittest_capi_inference_latency',
   install: get_option('install-test'),
   install_dir: unittest_install_dir
 )
-test('unittest_capi_inference_latency', unittest_capi_inference_latency, env: testenv)
+test('unittest_capi_inference_latency', unittest_capi_inference_latency, env: testenv, timeout: 100)
```

Comment:
let's add timeout to all test for later work.

Reply:
I updated the commit please check!